mix/qml/TransactionDialog.qml

```diff
@@ -67,112 +87,159 @@ Window {
 
 	function getItem()
 	{
-		var item = {
-			functionId: transactionDialog.functionId,
-			gas: transactionDialog.gas,
-			gasPrice: transactionDialog.gasPrice,
-			value: transactionDialog.transactionValue,
-			parameters: {}
+		var item;
+		if (!useTransactionDefaultValue)
+		{
+			item = {
+				functionId: transactionDialog.functionId,
+				gas: transactionDialog.gas,
+				gasPrice: transactionDialog.gasPrice,
+				value: transactionDialog.transactionValue,
+				parameters: {},
+				executeConstructor: isConstructorTransaction
+			};
 		}
+		else
+		{
+			item = TransactionHelper.defaultTransaction();
+			item.functionId = transactionDialog.functionId;
+			item.executeConstructor = isConstructorTransaction;
+		}
+
+		if (isConstructorTransaction)
+			item.functionId = qsTr("Constructor");
```

Comment:
This should include some special symbol, to avoid potential conflict with function name